reporting/src/main/resources/application.properties

```diff
@@ -1,3 +1,9 @@
 kafka.bootstrap.address=broker:29092
 reporting.subscriber=reportingSubscriber
-spring.data.mongodb.uri=mongodb://root:example@mongo:27017/admin
\ No newline at end of file
+#spring.data.mongodb.uri=mongodb://root:example@mongo:27017/admin
+spring.data.mongodb.uri=mongodb://localhost:27017/admin
+#managementms.hostname=http://management
+#managementms.port=8081
+managementms.hostname=http://localhost
+managementms.port=8081
+server.port=8084
```

Comment:
Need to setup settings for Docker.

- Mongo settings to be reverted to the commented connection string.
- managementms.port should be set to 8080
- managementms.hostname should be set to management
- server.port needs to be commented.